src/components/container/RoiContainer.tsx

```diff
@@ -36,29 +35,6 @@ export function RoiContainer({
     });
   });
 
-  useKbsGlobal([
-    {
-      shortcut: ['delete', 'backspace'],
-      handler: (event) => {
-        if (event.isTrusted) {
-          roiDispatch({ type: 'REMOVE_ROI', payload: undefined });
-        }
-      },
-    },
-
-    {
-      shortcut: ['Escape'],
-      handler: (event) => {
-        event.preventDefault();
-        if (event.isTrusted) {
-          roiDispatch({
-            type: 'CANCEL_ACTION',
-          });
-        }
-      },
-    },
```

Comment:
I forgot about this shortcut.

We wouldn't be able to reproduce this behaviour externally without exposing a `cancelAction`. So I think it would be good to add this action to the `useRoiActions` hook.